### DISCUSSION TITLE
hi i need help with this problem i have with github web page in google chrome.

### DISCUSSION THREAD

**Discussion by Darkrevengehack** [community/community#56378] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It looks like this:

it is not in desktop mode.
![Screenshot_20230525-025004](https://github.com/community/community/assets/103399387/25641546-40a3-41ba-979f-b03e025eddb9)

**Comment by plebgang** [comment]
Great
